src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -75,6 +75,274 @@ curie_map:
   GC_ID: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/GC_ID/"
   SNOMEDCT_2010_1_31: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/SNOMEDCT_2010_1_31/"
 
+extended_prefix_map:
```

Comment:
## Added to `mondo.sssom.config.yml`
I first was added this to #394 ([see comment](https://github.com/monarch-initiative/mondo-ingest/pull/394#discussion_r1456705175)), but I think this is better as a standalone PR.

I added it within `mondo.sssom.config.yml` as opposed to a standalone file. I have no idea however if this could have unintended side effects, so another consideration is that I could make a separate file for the EPM entirely and load that instead.